core/capabilities/webapi/target/launcher.go

```diff
@@ -0,0 +1,86 @@
+package target
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/connector"
+	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+)
+
+var _ job.ServiceCtx = &Launcher{}
+
+// Config is the configuration for the Target capability
+// Note that workflow executions have their own internal timeouts and retries set by the user
+// that are separate from this configuration
+type Config struct {
+	TimeoutMs  uint32 // Timeout in milliseconds from the capability to the gateway
+	RetryCount uint8  // Number of retries from the capability to the gateway
+}
+
+type Launcher struct {
+	gc         connector.GatewayConnector
+	targetCfg  Config
+	registry   core.CapabilitiesRegistry
+	lggr       logger.Logger
+	capability *Capability
+	handler    *Handler
+}
+
+func NewLauncher(gc connector.GatewayConnector, cfg string, registry core.CapabilitiesRegistry, lggr logger.Logger) (job.ServiceCtx, error) {
+	var targetCfg Config
+	if len(cfg) == 0 {
+		return nil, errors.New("config is empty")
+	}
+	// TODO: is config JSON for standard capabilities?
```

Comment:
No, it's TOML. It's part of a job spec (like relay config in OCR job specs).

Reply:
Ah no, this can be whatever the user defines -- it's stored as a string in the TOML spec

Reply:
That's a little weird. Why not standardize to nested TOML, like relay config?

Reply:
I'm not against that, I think it makes sense 👍